package.json

```diff
@@ -76,6 +76,7 @@
     "mongodb-extended-json": "^1.9.0",
     "mongoose": "^4.7.5",
     "morgan": "^1.8.1",
+    "nodemon": "^1.11.0",
```

Comment:
Can you actually add this to the dev dependencies? Those don't get installed in production. Since nodemon is a dev tool, it's not needed in any environment other than local.
`yarn add nodemon --dev`